src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -102,8 +110,54 @@ public void largeText() throws IOException {
                     )
                 )
             ).asString(),
```

Comment:
@ilyakharlamov please replace the `assertThat` with `Assertion`

Reply:
@victornoel fixed in 8915ec4